contracts/dtoken/src/common.rs

```diff
@@ -22,9 +22,9 @@ impl Contract {
 
     pub fn get_exchange_rate(&self, underlying_balance: WBalance) -> Balance {
         if self.token.total_supply == 0 {
-            return self.initial_exchange_rate;
+            return self.initial_exchange_rate * RATIO_DECIMALS;
```

Comment:
initial_exchange_rate -> should be Ratio by default. So there is no need to use multiplier - RATIO_DECIMALS